posthog/models/scheduled_change.py

```diff
@@ -1,17 +1,16 @@
 from django.db import models
-from django.utils import timezone
 
 
 class ScheduledChange(models.Model):
     class AllowedModels(models.TextChoices):
         FEATURE_FLAG = "FeatureFlag", "feature flag"
 
     id = models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")
-    record_id = models.IntegerField()
+    record_id: models.CharField = models.CharField(max_length=200)
     model_name: models.CharField = models.CharField(max_length=100, choices=AllowedModels.choices)
     payload: models.JSONField = models.JSONField(default=dict)
-    scheduled_at: models.DateTimeField = models.DateTimeField(default=timezone.now)
-    executed_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
+    scheduled_at: models.DateTimeField = models.DateTimeField(db_index=True)
```

Comment:
you can do this in one index, and ideally should: https://docs.djangoproject.com/en/4.1/ref/models/options/#django.db.models.Options.indexes